dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -434,6 +434,10 @@ ERROR_NON_FEATURE_CANNOT_INSTALL_TO_EXTENSION=CWWKF1298E: {0} is not a feature a
 ERROR_NON_FEATURE_CANNOT_INSTALL_TO_EXTENSION.explanation=Only features can be installed on a product extension.
 ERROR_NON_FEATURE_CANNOT_INSTALL_TO_EXTENSION.action=Run the installUtility install command again without specifying the extension.
 
+ERROR_FAILED_TO_RESOLVE_FEATURES_FOR_OPEN_LIBERTY=CWWKF1299E:Unable to obtain the following features: {0}. Ensure that the features are valid for Open Liberty.
```

Comment:
Change "Unable to obtain the following features:" to "The following features could not be obtained:".